### DISCUSSION TITLE
Upload PDF File Issue

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#8104] [Help] [answered]
when uploading .pdf file , it gives an error and wont upload the file

here is the code snippet , i only want to upload pdf files 

```
 Forms\Components\FileUpload::make('document')
                            ->acceptedFileTypes(['application/pdf'])
                            ->label(__('Document'))
```

![Screenshot from 2023-08-27 17-20-23](https://github.com/filamentphp/filament/assets/56083879/3662f93e-9f68-4e59-964a-7e7d5ee77e9f)

**Comment by pboivin** [comment]
Any errors in the browser console?

**Reply by Hegabovic** [reply]
no nothing

**Reply by pboivin** [reply]
Anything in `storage/logs/laravel.log` ?

**Reply by Hegabovic** [reply]
Failed to load resource: the server responded with a status of 422 (Unprocessable Content)

**Reply by HomaEEE** [reply]
Try to use Spatie Media Library plugin

**Reply by leandrocfe** [reply]
What is the file size?
Did you try to upload different files?

**Reply by HomaEEE** [reply]
You can load original file size (pdf don't compress). I use it for create pdf from single pages

**Reply by Hegabovic** [reply]
the thing is that ,when i tried to upload a 1 KB PDF it uploaded but the case is that i want to upload a 6 MB PDF file , how to do that ? i think this is considered a bug (i want to upload any size i want )

**Reply by HomaEEE** [reply]
It's settings of php - max_file_size and post_file, maybe Apache or nginx. Start from php.ini

**Reply by Hegabovic** [reply]
i think it is a bug in filament

**Reply by danharrin** [reply]
Doubt it. Check the Livewire config for the max size set there.

**Reply by Hegabovic** [reply]
here is my configuration 
###  .htaccess
```
  php_value upload_max_filesize 10M
  php_value post_max_size 10M
```
```
    'disks' => [
        'public' => [
            'driver' => 'local',
            'root' => storage_path('app/public'),
            'url' => env('APP_URL').'/storage',
            'visibility' => 'public',
            'size' => 10 * 1024 * 1024, // 10 megabytes
        ],
        // ...
    ],
```


and  still get the same uploading issue

**Reply by danharrin** [reply, marked as answer]
Livewire config
